google/cloud/aiplatform/models.py

```diff
@@ -1599,12 +1602,14 @@ def predict(
                 parameters=parameters,
                 timeout=timeout,
             )
+            metadata = json_format.MessageToDict(prediction_response._pb)["metadata"]
```

Comment:
Would `json_format.MessageToDict(prediction_response._pb.metadata)` work? (I'm not sure we need to convert the whole response. Also this might better deal with empty metadata.)